Components/Core/Inc/Command.hpp

```diff
@@ -21,7 +21,8 @@ enum GLOBAL_COMMANDS : uint8_t
     TASK_SPECIFIC_COMMAND,    // Runs a task specific command when given this object
     DATA_COMMAND,            // Data command, used to send data to a task. Target is stored in taskCommand
     CONTROL_ACTION,            // Control actions, used in Rocket State Machine, direct translation to RCU<->DMB Protocol
-    REQUEST_COMMAND            // Request command
+    REQUEST_COMMAND,            // Request command
+    RADIOHB_CHANGE_PERIOD,
```

Comment:
Add a comment, like `// Change Radio HB Period to Provided TaskCommand Period in Seconds`